hw/ip/kmac/data/kmac.prj.hjson

```diff
@@ -14,7 +14,7 @@
         life_stage:         "L1",
         design_stage:       "D2S",
         verification_stage: "V2",
-        dif_stage:          "S1",
+        dif_stage:          "S2",
```

Comment:
can you update the [checklist](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/dif/dif_kmac.md;l=1?q=dif_kmac.md&sq=&ss=opentitan%2Fopentitan) too?

Reply:
Done